yt_dlp/extractor/rplaylive.py

```diff
@@ -210,8 +215,24 @@ def _real_extract(self, url):
             'uploader_id': ('creatorOid', {str}),
             'tags': ('hashtags', lambda _, v: v[0] != '_'),
             'age_limit': (('hideContent', 'isAdultContent'), {lambda x: 18 if x else None}, any),
+            'location': ('location', {str}),
+            'view_count': ('views', {int}),
+            'like_count': ('likes', {int}),
             'live_status': ('isReplayContent', {lambda x: 'was_live' if x else None}),
         })
+        if self._preferred_lang:
+            translated_metainfo = traverse_obj(video_info, {
+                'title': ('multiLangTitle', self._preferred_lang, {str}),
+                'description': ('multiLangIntroText', self._preferred_lang, {str}),
+                'uploader': ('creatorInfo', 'multiLangNick', self._preferred_lang, {str}),
+            })
+            if ('title' not in translated_metainfo
+                    or 'description' not in translated_metainfo
+                    or 'uploader' not in translated_metainfo):
+                self.report_warning(
+                    f'Did not find translations for {self._preferred_lang} for all metadata. '
+                    'Metadata without translations will be in the original language', video_id)
```

Comment:
Better to make it clear which translation field is missing for the warning
```suggestion
            if missing := [key for key in ['title', 'description', 'uploader'] if key not in translated_metainfo]:
                self.report_warning(f'No translation for {", ".join(missing)}, will use original language')
```